src/lib.rs

```diff
@@ -86,8 +86,6 @@ extern crate log;
 #[macro_use]
 extern crate serde;
 #[macro_use]
-extern crate derivative;
```

Comment:
This crate should be `cfg`-ed to Windows, instead of removed.

Reply:
Brought it back with a cfg per your request.